### ISSUE TITLE
Is it possible to limit hosts included in accessibility testing?

### ISSUE THREAD

**Issue by WillGibson** [dequelabs/axe-core-gems#246]
We have some scenarios where the expected result is for the user to land on a different host.

E.g. we are developing and testing https://something.example.com and some scenarios end up on https://example.com.

Our scenarios then on fail accessibility tests because of issues on the main site which are not the concern of our team or the codebase we are working on.

This may be a how do I question or a feature request, but we would like to be able to limit the hosts that are accessibility tested with something like this...

https://www.deque.com/axe/core-documentation/api-documentation/#user-content-allowedorigins

Is that possible with the Ruby gem? I so is it documented?, If not, how do we do it?

**Comment by michael-siek** [comment]
Hey @WillGibson do you mind providing some code example on what particular issue you are having? Are you running an e2e workflow where the domain consistently changes?

**Comment by WillGibson** [comment]
I'm afraid that was from a "previous life" and I can't remember what the specific example was, but I think my original comment summarises it well enough.
